defactor-com/ui-kit
# Fix/lendborrow component does not accept formatted inputs 80

## Lendborrow component does not accept formatted inputs 

### What does this PR do?

- Resolve #80

### Steps to test

1. Run yarn storybook
2. Check changes

Before:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/b1b32138-1447-4e7d-991b-396b0ab8f307)

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/c62a6cc7-d326-49dd-8695-9c4f300bdcf5)


#### CheckList
